### PR TITLE
revert back some library version numbers to test ep merge integration…

### PR DESCRIPTION
**Pull request by chengjie8**
Test

### CODE REVIEW

File: svc-bgs-api/src/Gemfile [hunk 1]
```diff
@@ -7,8 +7,7 @@ gem 'cgi', '~> 0.3.6' # Address CVE-2021-41819
 gem 'bunny', '>= 2.13.0'
 gem 'activesupport', '~> 6.0'
 gem 'bgs_ext', git: 'https://github.com/department-of-veterans-affairs/bgs-ext.git', require: 'bgs'
-gem 'rexml', '>= 3.2.7'
```
Comment (lisac) [inline svc-bgs-api/src/Gemfile:10]: to double-check: is removing `rexml` here desired? looking at the PR #3080 that i believe we're attempting to partially revert, i don't see `rexml` as one of the changes.

**Review by lisac — APPROVED** [review]
seeing that the EP Merge CI tests pass when using this branch, i support merging it in. Great work, @chengjie8 !

File: svc-bgs-api/src/Gemfile (hunk 1 above)
Comment (chengjie8) [reply svc-bgs-api/src/Gemfile:10]: I was just testing to see which added library causes the ep-merge integration test to fail. It turned out that the rack library was the issue. I've added this back. Thanks for pointing it out @lisac.

Comment (chengjie8) [reply svc-bgs-api/src/Gemfile:10]: rexml is also breaking the integration test. Now i need to remove it.

**Pull request closed without merging**